.github/workflows/wine-ubuntu.yml

```diff
@@ -0,0 +1,27 @@
+name: Wine Ubuntu CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v2
+      - name: Compilation
+        run: |
+          sudo dpkg --add-architecture i386 && sudo apt update
+          cd wine-tkg-git 
+          sed -i 's/distro=""/distro="debuntu"/' customization.cfg
+          yes|./non-makepkg-build.sh
+      - name: Archive the artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: wine-tkg-build
```

Comment:
You are capturing the wrong folder, the correct one is `non-makepkg-builds`. For now the artifacts are 3.4Gb x)

Reply:
Fixed.

Reply:
Your suggestion points to empty dir 😄

Reply:
Fixed.